release notes/v0.34.0.md

```diff
@@ -0,0 +1,61 @@
+k6 v0.34.0 is here! :tada: It introduces the already announced `k6/execution` API and includes some more enhancements and a bunch of minor bug fixes.
+
+## New Features
+
+* k6/execution API
+
+Introduced a new module importable with the `k6/execution` path for fetching information about the execution state through a series of exported properties. It makes available information about VUs, Iterations and Scenarios.
+
+### How to use
+
+Accessing items iterating a dataset or getting them just once time cases are easier now. 
+
+```
+import exec from "k6/execution";
+import { SharedArray } from "k6/data";
+
+const data = new SharedArray("my dataset", function(){
+	// generate and return the data
+})
+
+export const options = {
+    scenarios :{
+        "use-all-the-data": {
+            executor: "shared-iterations",
+            vus: 100,
+            iterations: data.length,
+            maxDuration: "1h"
+        }
+    }
+}
+
+export default function() {
+    // this is unique even in the cloud
+    var item = data[exec.scenario.iterationInTest];
+    http.post("https://httpbin.test.k6.io/amazing-endpoint", item)
+}
+```
+
+**Docs**: [grafana/xk6-execution](https://github.com/grafana/xk6-execution)
+
+## Enhancements and UX improvements
+
+* Warn Windows users on importing dependencies as absolute paths. ([#2078](https://github.com/grafana/k6/pull/2078))
+* Pass setup data object into [handleSummary callback](https://k6.io/docs/results-visualization/end-of-test-summary/#handlesummary-callback). ([#2103](https://github.com/grafana/k6/pull/2103)) - thanks @SamuelJohnson01997
+
+## Breaking changes
+
+* The deprecated outputs Datadog and Kafka have been removed. ([#2081](https://github.com/grafana/k6/pull/2081))
+
+## Bugs fixed!
+
+* Use HTTP POST method instead of GET for pushing logs requests to Loki. ([#2100](https://github.com/grafana/k6/pull/2100))
+* Encode the BlacklistIP option using the CIDR notation. ([#2083](https://github.com/grafana/k6/pull/2083))
+* `ext.loadimpact` option has the same precedence as the script configuration during the consolidation process. ([#2099](https://github.com/grafana/k6/pull/2099))
+* Log tailing from cloud refreshes the unedrhood websocket connection in the case an unexpected error occurred. ([#2090](https://github.com/grafana/k6/pull/2090))
+
+## Internals
+
+* common.Bind has been replaced from a simpler and clearer API to get access to some useful information shared by registered [modules](https://pkg.go.dev/go.k6.io/k6@v0.33.1-0.20210831121908-a96a8abf4bb5/js/modules#Instance). ([#2108](https://github.com/grafana/k6/pull/2108))
```

Comment:
It wasn't replaced (removed) yet, so maybe:

```suggestion
* A simpler and clearer API has been added as an alternative to `common.Bind`, which also gives JS modules easy access to some [useful internal objects and runtime information](https://pkg.go.dev/go.k6.io/k6@v0.34.0/js/modules#Instance). ([#2108](https://github.com/grafana/k6/pull/2108))
```

Also maybe link to the final `v0.34.0` URL, though it will 404 until `pkg.go.dev` is updated. So maybe not? :man_shrugging:

Reply:
Keeping this as unresolved for maintaining the working link available https://pkg.go.dev/go.k6.io/k6@v0.33.1-0.20210831121908-a96a8abf4bb5/js/modules#Instance